pkg/upgrade/upgrade.go

```diff
@@ -0,0 +1,21 @@
+package upgrade
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/machinepool"
+)
+
+func NewUpgradeArgsFunction() func(cmd *cobra.Command, argv []string) error {
+	return func(cmd *cobra.Command, argv []string) error {
+		flags := cmd.Flags()
+		machinepoolID, _ := flags.GetString("machinepool")
```

Comment:
@Mischulee We shouldn't be ignoring the `error` here. Could you please add a check for the `err != nil` and return it from the function.

Reply:
Fixed.